### PR TITLE
login form error messages edited

### PR DESCRIPTION
**Pull request by AndroGenius-codes**
fixes issue 
#690 and #706 

login error solved

### CODE REVIEW

**Comment by AndroGenius-codes** [comment]
@jennydaman  @mairin can you help review my new commit. also i am seeing that a process in cypress run failed could you explain why for me

File: src/components/feed/CreateFeed/createfeed.scss [hunk 1]
```diff
@@ -263,3 +264,10 @@ Styles for pipelines
     }
   }
 }
+
+/*calender fix */
+.pf-c-input-group input:not([type=checkbox]):not([type=radio]), .pf-c-input-group textarea {
```
Comment (jennydaman) [inline src/components/feed/CreateFeed/createfeed.scss:269]: Please work on a branch from within your fork. You should be creating a branch per feature (per issue, per PR, ...)

File: src/pages/Dashboard/Dashboard.scss [hunk 2]
```diff
@@ -10,7 +11,14 @@
   }
 }
 @media screen and (min-width: $pf-global--breakpoint--xs) and (max-width: $pf-global--breakpoint--sm) {
+  .dashboard-carddisplay img:nth-last-of-type(-n + 2) {
+    display: none;
```
Comment (jennydaman) [inline src/pages/Dashboard/Dashboard.scss:15]: what is the purpose of these CSS?

**Review by jennydaman — CHANGES_REQUESTED** [review]
The red text error messages could probably be improved. Maybe a popup/toast would work here.

File: src/pages/Dashboard/Dashboard.scss (hunk 2 above)
Comment (AndroGenius-codes) [reply src/pages/Dashboard/Dashboard.scss:15]: > what is the purpose of these CSS?

no purpose .i was testing out something and i forgot to delete it.

**Pull request closed without merging**